api/resources/search_options.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+import json
+
+from flask import request
+from flask_restful import Resource, abort
+from sqlalchemy.orm.exc import NoResultFound
+
+from api import db
+from api.database.models import Applicant, Skill, Value
```

Comment:
im realizing importing `Applicant` may not be necessary here since we're not interacting with the `Applicant` model directly, though maybe the fact that we're doing SQL queries on it means it is necessary... just thinking 'out loud' and will try coming back to this later